docs/docs/connectors/twilio.mdx

```diff
@@ -59,3 +59,61 @@ twilio:
 
 Restart your Rasa server
 to make the new channel endpoint available for Twilio to send messages to.
+
+
+#### Receiving location data via twilio (whatsapp)
+
+You can receive location data from user from this channel you need to do the following.
+
+1. Make an intent name locationData and two entities/slots one for Latitude and one for Longitude respectively.
+
+```yaml-rasa title="domain.yml"
+intents:
+  - locationData
+slots:
+   Latitude:
+     type: text
+     mappings:
+     - type: custom
+
+   Longitude:
+     type: text
+     mappings:
+     - type: custom
+
+entities:
+  - Latitude
+  - Longitude
+
+actions:
+  - action_location_data
+```
+
+2. Once user send the location message it will Trigger the locationData intent you can later handle everything in custom actions.
```

Comment:
I think this custom action adds complexity to your PR and the documentation. It isn't necessary either since slot mapping will set the slots. Do you think it'll be a good idea to remove it?

Reply:
yeah ill think remove it